data/index.json

```diff
@@ -130,6 +130,35 @@
                     "delim": "\t"
                 }
             }
+        },
+        {
+            "id": "admins",
+            "name": "Geo Administrative Levels",
+            "url": "https://osf.io/y3q8e/download",
+            "checksum": "4a1fd5f3891eaf5deffa3c69c8c18fc3b254847be327ec6961b4a13a180ae2ef",
+            "webpage": "https://osf.io/y3q8e/",
```

Comment:
https://gitlab.com/ViDA-NYU/auctus/datamart-geo is probably a better link, I am only using OSF for storage and there are no details there